src/components/Node/Node.js

```diff
@@ -380,6 +381,9 @@ const ThemedNode = styled.div`
 const ThemedThumbnail = styled.div`
   width: ${({ theme }) => theme.thumbWidth}px;
   height: ${({ theme }) => theme.thumbHeight}px;
+  .imgMore {
+    display: ${({ theme }) => theme.imageCounter};
```

Comment:
let's call it `thumbCounterDisplay` plz

Reply:
sure, how about _thumbnaillCounter_

Reply:
definitely not with 2 `L`s but yeah, if you want you can rename all the thumb to thumbnail, I'm keen to preserve the name of the css prop you're setting 👍